README.md

```diff
@@ -1 +1,150 @@
-# capstone-project
\ No newline at end of file
+# capstone-project
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <br/>
+  <h3><b>Portolio Mobile first set up</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖capstone-project <a name="about-project"></a>
+
+
+**capstone-project** This is a project about my portfolio webpage that begins with mobile screen sizes to desktops screen sizes.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML,CSS</summary>
+  <summary>Linters</summary>
+  <summary>Github</summary>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **Adding pictures**
+- **Using flex**
+- **Side bar menu**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+This is the [live demo link](https://firstpj.github.io/capstone-project/)
+
+
+## 🚶‍♂️ Walkthrough <a name="walkthrough"></a>
+
+-[Walkthrough Link](https://https://www.loom.com/share/671307d7da33458da2b8cc6f51252079)
```

Comment:
- [ ] Despite your efforts to implement suggested changes, the above link to the Project presentation is still inaccessible 😢 I would suggest that you check again the link for correctness. If it is correct, that means you have not granted access for it to be accessed or viewed by everyone. That is why I suggest that you can ensure that that access is granted too 😎